plugins/vertica/database_credentials_test.go

```diff
@@ -0,0 +1,32 @@
+package vertica
```

Comment:
Let's please add a test for the importer as well.

Reply:
added